desktop/Makefile

```diff
@@ -158,13 +158,11 @@ node_modules/%:
 node_modules: package.json
 	@$(NPM) prune
 	@$(NPM) install
-	@touch node_modules
```

Comment:
I'm not so sure that this is the right solution - it avoids an error in the CircleCI build which I believe stems from an issue with [node-touch](https://github.com/isaacs/node-touch/issues/8). My worry here is that there is good reason for `touch`ing `node_modules` at the end of these make targets. @johngodley, it looks like these were added in the initial commit - are they safe to remove like this?